pkg/ui/workspaces/db-console/src/views/reports/containers/debug/index.tsx

```diff
@@ -264,12 +264,12 @@ export default function Debug() {
       {disable_kv_level_advanced_debug && (
         <section className="section">
           <InlineAlert
-            title="Some advanced debug options are not available on secondary tenants."
+            title="Some advanced debug options are not available on virtual clusters."
             intent="warning"
             message={
               <span>
                 To access additional advanced debug options, please login using
-                system tenant credentials.
+                the system interface.
```

Comment:
This is an improvement. But, I worry that users might not know what "the system interface" is. Is that a different login page.  Perhaps "please login with a user that has system-level access" but even that isn't too clear to me.  This is worth thinking about as we go.

Reply:
Agreed - let's push this until later.